webextension/background.js

```diff
@@ -274,6 +274,14 @@ const Config = (function() {
       return timeout;
     }
 
+    get testingMode() {
+      return this._testingMode;
+    }
+
+    get reportLanding() {
+      return "https://blipz-experiment-issues.herokuapp.com/new";
```

Comment:
Hard coded is probs fine for now, but as soon as we get into serious testing mode, or further iterations of the experiment I'd want a pref here so we can point it at a mochitest server or whatever. Something like `extensions.blipz.reportEndpoint`